api/cloud/oc_cloud_store.c

```diff
@@ -300,6 +300,8 @@ cloud_store_initialize(oc_cloud_store_t *store)
   cloud_set_string(&store->uid, NULL, 0);
   cloud_set_string(&store->access_token, NULL, 0);
   cloud_set_string(&store->refresh_token, NULL, 0);
+#define DEFAULT_CLOUD_SID "00000000-0000-0000-0000-000000000000"
+  cloud_set_string(&store->sid, DEFAULT_CLOUD_SID, strlen(DEFAULT_CLOUD_SID));
   cloud_set_string(&store->sid, NULL, 0);
```

Comment:
In my opinion this line should be removed.

Reply:
Yes, should have been replaced, instead I've kept it by mistake. I'll fix it in the morning.